components/bounty/pull-and-proposal-hero/view.tsx

```diff
@@ -0,0 +1,119 @@
+import BigNumber from "bignumber.js";
+import { useTranslation } from "next-i18next";
+
+import ArrowLeft from "assets/icons/arrow-left";
+
+import AvatarOrIdenticon from "components/avatar-or-identicon";
+import PriceConversor from "components/bounty/bounty-hero/price-conversor/controller";
+import Button from "components/button";
+import CustomContainer from "components/custom-container";
+import DateLabel from "components/date-label";
+import GithubInfo from "components/github-info";
+
+import { truncateAddress } from "helpers/truncate-address";
+
+interface PullAndProposalHeroViewPRops {
+  contractId: number;
+  githubLogin: string;
+  createdAt: Date;
+  creatorAddress: string;
+  issueTitle: string;
+  issueGithubId: string;
+  issueAmount: BigNumber;
+  transactionalTokenSymbol: string;
+  isProposal: boolean;
+  onBackClick: () => void;
+}
+
+export default function PullAndProposalHeroView({
+  contractId,
+  githubLogin,
+  createdAt,
+  creatorAddress,
+  issueTitle,
+  issueGithubId,
+  isProposal,
+  issueAmount,
+  transactionalTokenSymbol,
+  onBackClick,
+}: PullAndProposalHeroViewPRops) {
+  const { t } = useTranslation([isProposal ? "proposal" : "pull-request", "common"]);
+
+  return (
+    <div className="w-100 border-bottom border-gray-800">
+      <CustomContainer className="p-3">
+        <div className="row align-items-center mb-4">
+          <div className="col-auto">
+            <Button
+              className="rounded-circle p-1 text-white not-svg"
+              onClick={onBackClick}
+              outline
+            >
+              <ArrowLeft width={10} height={10} />
+            </Button>
+          </div>
+
+          <div className="col px-0">
+            <span className="me-2 text-gray-500 caption-large font-weight-medium">
+              #{issueGithubId}
+            </span>
+            <span className="text-white caption-large font-weight-medium text-capitalize">
+              {issueTitle}
+            </span>
+          </div>
+        </div>
+
+        <div className="row align-items-center mb-4">
+          <div className="col">
+            <div className="row align-items-center">
+              <span className="caption-large text-white text-capitalize">
+                {t("title")}
+
+                <span className="ml-1 text-gray-500 font-weight-medium">
+                  #{+(contractId || 0) + 1}
```

Comment:
contract id should be correct, why are we adding 1 to it? :o

Reply:
@moshmage it's just for viewing purpose, if we don't add 1 it will be shown as `#0`
![image](https://github.com/layerx-labs/webnetwork/assets/9003126/9b0c0cca-83dc-4714-815a-0f727866feeb)